docs/introduction.md

```diff
@@ -42,7 +44,34 @@ The final version of the previous execution state remains available through a le
 
 ## Flow
 
-Flow was designed to provide a blockchain that can scale while preserving composability in a single blockchain state. This is achieved by introducing several [node roles](#nodes), each with their own trade-off for the [scalability trilemma](https://aakash-111.medium.com/the-scalability-trilemma-in-blockchain-75fb57f646df).
+Most current blockchains are built as a homogenous system comprised of *full nodes*.
+Each full node is expected to collect and choose the transactions to be included in the next block, execute the block, come to a consensus over the output of the block with other full nodes, and finally sign the block and append it to the chain.
```

Comment:
This is not entirely accurate. Every full node collect and propagate transactions.

However, for proof-of-work blockchains, only miners actually create blocks these days; normal nodes can no longer compete with the massive computation power of mining companies.

For proof-of-stake blockchains, which is a more apt comparison for Flow, one leader is chosen for each consensus round, and that leader node is the only one proposing a block with transactions.

Also, the signing part doesn't make sense. In proof-of-work, there is no signing. In proof-of-stake, the proposer signs his block proposal before it is validated by everyone. Other nodes who validated it will sign it as well as a "vote" for the proposal, and if it collects enough votes, the block is finalized and stored along with all of the signatures.

Reply:
Noted. This whole paragraph now looks pretty wrong, given the background :) Should I perhaps remove it entirely?

"only miners actually create blocks these days" - is it because their role/status in the system is different, or is it just a manifestation of their computational power?

For proof-of-stake blockchains - are all of the nodes otherwise the same? E.g. there's 10 identical nodes, and of them one is chosen as the leader that will propose the block, remaining ones are (potential?) validators?